pkg/resource/model_quality_job_definition/sdk.go

```diff
@@ -605,8 +605,7 @@ func (rm *resourceManager) sdkUpdate(
 	latest *resource,
 	delta *ackcompare.Delta,
 ) (*resource, error) {
-	// TODO(jaypipes): Figure this out...
-	return nil, ackerr.NotImplemented
+	return nil, ackerr.NewTerminalError(ackerr.NotImplemented)
```

Comment:
@RedbackThomson @A-Hilaly Not implemented does not give sufficient information to the customer. What are your thoughts to reword it  `update not implemented` ?